src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -36,6 +38,8 @@ open import Relation.Binary
 import Relation.Binary.Consequences as BC
 open import Relation.Binary.PropositionalEquality
 import Relation.Binary.Properties.Poset as PosetProperties
+open import Relation.Nullary using (yes; no)
+import Relation.Binary.Reasoning.Setoid as ReasonSetoid
```

Comment:
We usually name this `SetoidReasoning` not `ReasonSetoid`

Reply:
I renamed it.